sdk/go/internal/engineconn/http/http.go

```diff
@@ -0,0 +1,37 @@
+package http
```

Comment:
/cc @vito 

There's some overlap with #3671

I picked `http` because that's what the Python SDK currently requires -- but we can update that

Reply:
HTTP works too :+1: 

I went with TCP early on when it wasn't clear whether I'd need to switch everything to gRPC, but now that it's gRPC-over-HTTP it doesn't really matter. I can just rebase on this. Besides, the TCP connection was just for local testing by hand.